src/components/views/rooms/RoomHeader.tsx

```diff
@@ -195,7 +195,10 @@ const VideoCallButton: FC<VideoCallButtonProps> = ({ room, busy, setBusy, behavi
         menu = <IconizedContextMenu {...aboveLeftOf(buttonRect)} onFinished={closeMenu}>
             <IconizedContextMenuOptionList>
                 <IconizedContextMenuOption label={_t("Video call (Jitsi)")} onClick={onJitsiClick} />
-                <IconizedContextMenuOption label={_t("Video call (Element Call)")} onClick={onElementClick} />
+                <IconizedContextMenuOption
+                    label={_t("Video call (%(brand)s Call)", { brand: SdkConfig.get("brand") })}
```

Comment:
What if someone wanted to even change the `Call` part?

Reply:
This feels like a product decision, so I've [asked about it internally](https://matrix.to/#/!gMLaQlBnlkoVcEZXnA:element.io/$1Ld3AgwHalm_PDSzHXvYTNK_zYxNTth5tnyeqEILf6E?via=element.io&via=matrix.org&via=robin.town)